### ISSUE TITLE
Stats: Referrers: Change how we Label Android Referrers

### ISSUE THREAD

**Issue by thehenrybyrd** [Automattic/wp-calypso#35367]
In Stats, we currently group views from _any_ Android app under "Wordpress Android App". However, most of these views are _not_ from the WordPress Android app, and are in fact from other Android apps such as Gmail or Twitter apps.

#### Steps to reproduce
1. Send yourself an email with a link to one of your blogs.
2. Check your email in the Gmail app on an Android device.
3. Click that link, it should take you to your blog in Gmail's in-app browser.
4. Head to Stats in Calypso, and note that you have a view from the referrer “Wordpress Android App” with the link `http://android-app//com.google.android.gm`

#### What I expected

Checking my email should result in a referrer like "Gmail" or "Email".

#### What happened instead

The referrer is "Wordpress Android App”, which leads me to believe the site was viewed directly from the WordPress Android App. This is not the case.

![screen_shot_on_2019-08-14_at_10_35_41_720](https://user-images.githubusercontent.com/15107387/63010271-bdc5c880-be8e-11e9-9535-8ac8ed5a9d65.png)

There are a number of other stats which come from Android devices, but are labeled as "Wordpress Android App", incorrectly. This can cause confusion about where these stats come from.
This is a poor stats grouping, and should be labeled differently.

Grouping was created in: r178394-wpcom
`case 'android-app' : return 'WordPress Android App';`

#### Browser / OS version

Chrome 75.0.3770.142  on MacOS 10.14.5
-and- 
Gmail App on Nokia 2, Android 7.1.1

#### Context / Source
A user came wondering about the meaning of stats coming from "Wordpress Android App".
 #user-report

**Comment by thehenrybyrd** [comment]
I ran into this again with another user of the WordPress app, confused about why so many views were coming from "WordPress Android App".

**Comment by thehenrybyrd** [comment]
Here's another example, from another conversation with a confused user. This one thought they'd been hacked and someone else was using the WordPress app to edit their site, which is not the case.
None of these referrers are WordPress.
<img width="1044" alt="WordPress Android App" src="https://user-images.githubusercontent.com/15107387/79682852-01df4900-8226-11ea-9e19-ba23cdaa5e92.png">

**Comment by thehenrybyrd** [comment]
Ran into this again today with a user who thought these were spam, since the links lead nowhere.

**Comment by sudiptog81** [comment]
Seems like a core WP issue and not `wp-calypso` specific. I recorded an odd count of visitors (> 1,000) referred by  "Wordpress Android App" which in fact were triggered from other Android (API Level 28) applications like GMail, K-9 Mail, WhatsApp etc.

**Comment by planarvoid** [comment]
As far as I see this is skipping the WordPress Android App completely. Would the simplest solution maybe be to replace this: 
`case 'android-app' : return 'WordPress Android App'` with something like this `case 'android-app' : return 'Android Apps';`? I also don't think that the Android Application ID (= the `com.google.android.gm` thing) is a very useful field to show to the user since it's not very readable (even though it's unique).

**Comment by thehenrybyrd** [comment]
@planarvoid I agree - returning 'Android Apps' instead of 'WordPress Android App' would reduce confusion here. 
It would be nice if we were able to pull in the details of which app (Gmail, WhatsApp, etc as @sudiptog81 mentions) as well, as this would be more clear to users. They could then know that someone arrived at their site from the Gmail app, for instance. Just the change to 'Android Apps' would be a really good start though.

**Comment by brunoleal2003** [comment]
Hello, everyone. My name is Bruno. I am editor of a brazilian history popularization site. Well, I notice the same issue: a descrepancy between the Jetpack Stats and the Google Analytics Stats. There will always be a discrepancy. It's normal. But the discrepancy on my website is anything but normal. Jetpack/WP is registering 10x more pageviews than Google Analytics. This usually happens after a new post. I took a look at the Jetpack statistics panel and in the "References" tab, I also discovered that the access discrepancy has a name: "WordPress Android APP”, or com.google.android.googlequicksearchbox. I read you replys, but I'm still confuse on what you find it. 

(1) This page views coming from this source are real or not? 

(2) And if they are real, why does GA not count them?

(3) If they are real, they consume or not my server resources?

**Comment by rinazrina** [comment]
I got a user who would like to understand the exact source of referrers under "WordPress Android App," as they got a lot of traffic from that. I clarified that they're coming from Android Apps such as Google Quick Search box, however, the word "WordPress" confused them. I second that changing the Referrers group to "Android apps" would make much more sense.

**Comment by igotdes** [comment]
Another user asked about this. They were wondering about the sudden drop in their stats, and the biggest difference seems to be from the “WordPress Android App” referrer:

![Markup on 2022-06-09 at 09:59:02](https://user-images.githubusercontent.com/74066241/172748047-3b456d42-ac09-464e-9415-c43f5e7b1e60.png)

**Comment by Jaccuse** [comment]
I chatted with a user who would basically like for these stats to be a bit easier to read. They believe it's a little time consuming to look closer and read through the URLs to determine if something came from the actual WordPress.com app or Google's quick search. They understand this is low priority, just putting this here since they did request it. 37715224-hc for reference.

**Comment by zdenys** [comment]
5974543-zd-woothemes

**Comment by filipanoscampos** [comment]
6284216-zd-woothemes

This ticket is also related to this. It caused some confusion.

**Comment by khristiansnyder** [comment]
I ran into this today investigating some stats issues. The labeling "com.google.android.googlequicksearchbox" under WordPress Android App is quite confusing.

**Comment by donalirl** [comment]
Additional confusion in #8826755-zen

**Comment by donalirl** [comment]
To minimize confusion, this is now documented here: https://wordpress.com/support/stats/understand-your-sites-traffic/#referrers

> The “WordPress Android App” referrer encompasses traffic from any Android app, and can include traffic from Google Search, Gmail, Telegram, and many other apps someone may have used to find your website on their Android phone.

If the label changes, we should update the doc again.

**Comment by jartes** [comment]
Noting that this is still happening, as I saw it while supporting a user on 9538853-zd-a8c
